types/js-yaml/index.d.ts

```diff
@@ -48,6 +48,28 @@ export interface LoadOptions {
 	schema?: SchemaDefinition;
 	/** compatibility with JSON.parse behaviour. */
 	json?: boolean;
+	/** listener for parse events */
+	listener?: (eventType: EventType, state: State) => void;
```

Comment:
`listener` is called with a `this` context of type `State`:
```suggestion
	listener?(this: State, eventType: EventType, state: State): void;
```